src/ceph.in

```diff
@@ -712,7 +712,7 @@ def main():
         return 1
     except Exception as e:
         print >> sys.stderr, 'Error connecting to cluster: {0}'.\
-            format(e.__class__.__name__)
+             format(e.message)
```

Comment:
this is a bit dangerous because `.message` is going away soon (it is deprecated). Only `.args` is ensured to be kept around.